examples/AutoScrollExample.js

```diff
@@ -23,8 +23,12 @@ class AutoScrollExample extends React.Component {
     };
 
     this.columns = [];
-    const cellRenderer = ({ columnKey, rowIndex }) =>
-      (<div className='autoScrollCell'> {rowIndex}, {columnKey} </div>);
+    const cellRenderer = ({ columnKey, rowIndex }) => (
+      <div className="autoScrollCell">
+        {' '}
```

Comment:
Should we correct this manually?

Reply:
The transformation done by prettier is actually correct and has no net change.
But yeah, we can remove it.